src/tracks/playabletrack/notetrack/ui/NoteTrackVZoomHandle.cpp

```diff
@@ -256,8 +249,7 @@ void NoteTrackVRulerMenuTable::OnZoom( int iZoomCode ){
 BEGIN_POPUP_MENU(NoteTrackVRulerMenuTable)
 
    // Accelerators only if zooming enabled.
-   bool bVZoom;
-   gPrefs->Read(wxT("/GUI/VerticalZooming"), &bVZoom, false);
+   bool bVZoom = true;
```

Comment:
I think you didn't complete your intended work:  this context menu items still mention the mouse button shortcuts, but you removed those for waveform and spectrum rulers.